proto/users/users.proto

```diff
@@ -26,6 +26,7 @@ message User {
     string fullName = 5;
     repeated string hierarchy = 6;
     string hierarchyFlat = 7;
+    string adfsId = 8;
```

Comment:
What is the difference between id and adfsId? And where is it in use?

Reply:
The adfsId contains the mail of the user whereas the id contains random letters and numbers that identify the DB document.
To achieve the user T, you need to extract it from the adfsId. The user T is in use in the hi-service.